src/Sylius/Bundle/ApiBundle/spec/Doctrine/ORM/QueryExtension/Shop/Order/ShopUserBasedExtensionSpec.php

```diff
@@ -180,7 +180,7 @@ function it_applies_conditions_to_item(
         ShopUserInterface $user,
         CustomerInterface $customer,
         Expr $expr,
-        Expr\Func $exprEq,
+        Expr\Comparison $exprEq,
```

Comment:
There two `eq` comparison in this file, above one renamed to `exprComparison`, but this one doesn't.

```suggestion
        Expr\Comparison $exprComparison,
```

Same for `src/Sylius/Bundle/ApiBundle/spec/Doctrine/ORM/QueryExtension/Shop/Order/StateBasedExtensionSpec.php` file further in the PR.

Let's sync naming scheme and either keep it named after comparison type like `$exprEq` or general `$exprComparison`.